cmd/cvc-operator/controller/cstorvolumeclaim.go

```diff
@@ -375,7 +384,10 @@ func (c *CVCController) distributeCVRs(
 	for count, pool := range usablePoolList.Items {
```

Comment:
`	poolList, err := listCStorPools(cspcName, claim.Spec.ReplicaCount)`
Just list the cstor pools in listCStorPools, and have the check on count after getting usable pool list against pendingReplicaCount rather than spec.ReplicaCount

Reply:
changing name of ReplicaCount?

Reply:
'if else' code at `	if claim.Spec.CstorVolumeSource != "" {` need to be rewritten

Reply:
added the check other two comments will be fixed while moving the code to cstor-operators repo.